docs/Install-Guide/compiling-rpms.md

```diff
@@ -81,71 +81,73 @@ These steps are for both Fedora and RHEL/CentOS. At the end you'll have the comp
 
-		$ sudo yum -y --disablerepo=rhs* --enablerepo=*optional-rpms install git autoconf \
-		  automake bison dos2unix flex fuse-devel glib2-devel libaio-devel \
-		  libattr-devel libibverbs-devel librdmacm-devel libtool libxml2-devel lvm2-devel make \
-		  openssl-devel pkgconfig pyliblzma python-devel python-eventlet python-netifaces \
-		  python-paste-deploy python-simplejson python-sphinx python-webob pyxattr readline-devel \
-		  rpm-build systemtap-sdt-devel tar libcmocka-devel
+		# sudo yum -y --disablerepo=rhs* --enablerepo=*optional-rpms install git autoconf \
+		    automake bison dos2unix flex fuse-devel glib2-devel libaio-devel \
+		    libattr-devel libibverbs-devel librdmacm-devel libtool libxml2-devel lvm2-devel make \
+		    openssl-devel pkgconfig pyliblzma python-devel python-eventlet python-netifaces \
+		    python-paste-deploy python-simplejson python-sphinx python-webob pyxattr readline-devel \
+		    rpm-build systemtap-sdt-devel tar libcmocka-devel
 
 2. Clone the GlusterFS git repository
 
-    	$ git clone `[`git://git.gluster.org/glusterfs`](git://git.gluster.org/glusterfs)
-		$ cd glusterfs
+    	# git clone `[`git://git.gluster.org/glusterfs`](git://git.gluster.org/glusterfs)
+		# cd glusterfs
 
 3. Choose which branch to compile
 
-If you want to compile the latest development code, you can skip this step and go on to the next one. :)
+    If you want to compile the latest development code, you can skip this step and go on to the next one. :)
 
-If instead you want to compile the code for a specific release of GlusterFS (such as v3.4), get the list of release names here:
+    If instead you want to compile the code for a specific release of GlusterFS (such as v3.4), get the list of release names here:
 
-		$ git branch -a | grep release
-		  remotes/origin/release-2.0
-		  remotes/origin/release-3.0
-		  remotes/origin/release-3.1
-		  remotes/origin/release-3.2
-		  remotes/origin/release-3.3
-		  remotes/origin/release-3.4
-		  remotes/origin/release-3.5
+		# git branch -a | grep release
+		remotes/origin/release-2.0
+		remotes/origin/release-3.0
+		remotes/origin/release-3.1
+		remotes/origin/release-3.2
+		remotes/origin/release-3.3
+		remotes/origin/release-3.4
+		remotes/origin/release-3.5
 
-Then switch to the correct release using the git "checkout" command, and the name of the release after the "remotes/origin/" bit from the list above:
+    Then switch to the correct release using the git "checkout" command, and the name of the release after the "remotes/origin/" bit from the list above:
 
-		$ git checkout release-3.4
+		# git checkout release-3.4
 
-**NOTE -** The CentOS 5.x instructions have only been tested for the master branch in GlusterFS git. It is unknown (yet) if they work for branches older then release-3.5.
+    **NOTE -** The CentOS 5.x instructions have only been tested for the master branch in GlusterFS git. It is unknown (yet) if they work for branches older then release-3.5.
```

Comment:
this is also sort of obsolete.

Reply:
Same as above. Content changes in separate PR.